be/src/services/entity.service.ts

```diff
@@ -126,6 +128,106 @@ export class EntityService {
                 }
             })
     }
+
+    public async listEntities(query: ListQueryModel, id?: string | ObjectId): Promise<ListingResult<Entity>> {
+        const {options, filter} = this.transformToMongoDBFilterOption(query, id);
+        // @ts-ignore
+        const dbos: Entity[] = (await EntityService.collection().find(filter, options).toArray()) as Entity[];
+        return {
+            offset: query.offset,
+            rows: dbos.length,
+            totalItems: 0, // TODO
+            items: dbos
+        };
+    }
+
+    private escapeRegExp(string: string): string {
+        return string.replace(/[.*+?^${}()|[\]\\]/g, '');
+    }
+
+    // eslint-disable-next-line max-len
+    private transformToMongoDBFilterOption(query?: ListQueryModel, id?: string | ObjectId): {options: FindOptions; filter: Filter<Entity>} {
+        const options: FindOptions = {};
+        const filter: Filter<Entity> = {};
+
+        if (!!id) {
+            // @ts-ignore
+            filter.vocabulary = new ObjectId(id);
+        }
+
+        if (!!query?.type){
+            filter.type = query.type
```

Comment:
There is not a check if the type is one of the values used by the enum `TagType`, please include a check for this and throw a 400 error msg